src/App.tsx

```diff
@@ -1,14 +1,56 @@
 /* eslint-disable max-len */
```

Comment:
you should remove it from your file. You should fix linting issues instead of disabling them